serf/snapshot.go

```diff
@@ -102,7 +103,7 @@ func NewSnapshotter(path string,
 	logger *log.Logger,
 	clock *LamportClock,
 	outCh chan<- Event,
-	shutdownCh <-chan struct{}) (chan<- Event, *Snapshotter, error) {
+	shutdownCh <-chan struct{}, metricLabels []metrics.Label) (chan<- Event, *Snapshotter, error) {
```

Comment:
This breaks compatibility for the library function. We'll need a new function to pass the labels

Reply:
fixed